MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/TMA-troubleshooting/TMA-Troubleshooting-OSS-Fails-to-Turn-On-index.rst

```diff
@@ -0,0 +1,181 @@
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+- Chiller(s) light in the EUI OSS screen is red. **"OSS"**, **"observation"** and **"cooling"** lights are red too. On the alarm banner on the bottom, you can read **"LOCAL: COOLING SYSTEM"**
+
+.. figure:: ./_static/OSS_general_view.png
+   :name: OSS general view 
+
+
+
+
+
+
+The **Alarm History** screen shows
+
+.. figure:: ./_static/alarm_history.png
+    :name:  Alarm History
+
+.. _Title-of-Troubleshooting-Procedure-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. todo::
+   Make sure everything is in a safe or idle state before troubleshooting. Describe relevant safety steps if necessary.
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+
+
+
+.. _Title-of-Troubleshooting-Procedure-Critical-Step-1:
+
+ 1. Go to Level 1 Machinery Room and bring a Philips screwdriver with you. The Machinery Room is in the left door in the picture below.
+
+
+
+
+.. figure:: ./_static/level-1.png
+    :name: level 1
+
+
+
+
+2. The screens in the front panel of one or both of the chiller cabinets would display an alarm. On the top, the red band may say: **“STATUS: NOT ACKNOWLEDGED”** or **“STATUS: ALARM”**
+
+
+
+
+.. figure:: ./_static/chiller_2.jpeg
+    :name: chiller 2
+
+
+
+
+.. figure:: ./_static/alarm_menu.jpeg  
+    :name: alarm menu
+
+
+
+
+
+The large grey box on the cabinet's side (purple arrow) shows a blinking red light and an interlock in the screen.
+
+
+
+.. figure:: ./_static/grey_box_cabinets.jpeg
+    :name: grey box cabinets
+
+    .. figure:: ./_static/interlock_screens.jpeg
+        :name: interlock screens
+
+3. There are two reset buttons inside the chiller cabinet that needs to be pressed. From the side of the cabinet, use the screwdriver to remove two screws that are located on top. This will allow you to remove the cover panel.
+ Each cabinet has two panels. In chiller #2 the reset buttons are within the furthermost panel (Image below), while in chiller#1 they are found under the panel closer to the front face of the chiller.
+
+.. figure:: ./_static/furthermost_panel.jpeg
+    :name: remove panel
+
+    .. figure:: ./_static/Hand.jpeg
```

Comment:
Same with this image that doesn´t show in the html. It´s because the .. are not at the beginning of the line and the interpreter doesn´t understand it´s an image.

Reply:
Fixed I can see all the images. Please confirm.